### ISSUE TITLE
AttributeError: module 'pandas.core.common' has no attribute 'in_qtconsole'

### ISSUE THREAD

**Issue by dtherrick** [sassoftware/python-swat#40]
Running `python-swat` v1.5.0 with `pandas` v0.24.1 in a Jupyter notebook.
Any call to display formatted data in the notebook (for example, `conn.caslibinfo()`) throws the attribute error above.

In present state, the easy workaround is to force Pandas to install version 0.23.4. Doing this and rerunning the notebook restores the expected formatting.

Short term fix would be to note in documentation and setup files that Pandas versions 0.16.0 through 0.23.4 are supported. Longer term fix would be to remove the dependence on `in_qtconsole`.

**Comment by bensloane** [comment]
See https://github.com/sassoftware/python-swat/commit/1972603b9e589e08bbe62a7c87abd1d326f1f2bc this was addressed. 

You can also do the following for a temporary fix, if you don't want to mess with versioning or clone with latest updates, you can run this in jupyter.

```python 
get_ipython().config.get('IPKernelApp', {})['parent_appname'] = ""  
```

**Comment by dtherrick** [comment]
Thanks @bensloane - I'll try it with the swat-dev release as well as bumping back up to pandas 0.24.1. Good news is I'm doing all of this in a container so it's very easy to play around with different versions.

**Comment by kesmit13** [comment]
The fix was made in 1972603b9e589e08bbe62a7c87abd1d326f1f2bc.  A new release should be out soon.
